src/Abp/WebHooks/WebHookWorkItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Net;
+using Abp.Domain.Entities;
+using Abp.Domain.Entities.Auditing;
+
+namespace Abp.WebHooks
+{
+    /// <summary>
+    /// Table for store webhook work items. Each item stores web hook send attempt of <see cref="WebHookInfo"/> to subscribed users
+    /// </summary>
+    [Table("AbpWebHookWorkItems")]
+    public class WebHookWorkItem : Entity<Guid>, IMayHaveTenant, IHasCreationTime, IHasModificationTime, ISoftDelete
+    {
+        /// <summary>
+        /// <see cref="WebHookInfo"/> foreign id 
+        /// </summary>
+        public Guid WebHookId { get; set; }
+
+        /// <summary>
+        /// <see cref="WebHookSubscription"/> foreign id 
+        /// </summary>
+        public Guid WebHookSubscriptionId { get; set; }
```

Comment:
Same here,  add `WebHookSubscription` reference?

Reply:
Actually, subscriptions are stored in the host database but work items can be also stored in the tenant database. Because no need to loop each tenant databases to get it's a subscription. If you have a thousand tenant opening and closing connection causes a huge performance loss.
At the same time, most of the time no need to get webhook attempt of another tenant(almost never need it) so all tenants can store their webhook attempts and webhook info in their database.